### ISSUE TITLE
Make tables work with Google Translate

### ISSUE THREAD

**Issue by chelseanbr** [h2oai/wave#1749]
#### Wave SDK Version, OS

H2O Wave v0.22+ (v0.22.0 - v0.24.1) on macOS, Chromium browser (Google Chrome, Brave)
- Perhaps related to the v0.19 table redesign (?) when tables started having gray headers

#### Actual behavior

In Wave **v0.22+ (v0.22.0 - v0.24.1)**, tables are no longer compatible with the Google Translate browser extension. **The screen goes blank** when trying to translate the page, as shown below:

![wave_24](https://user-images.githubusercontent.com/29825155/206808275-a0aa2c24-c7cb-4fca-9e77-19ef7a632311.gif)

___
> Please note the example is already in English, but it does not matter what language is being translated to/from. The screen goes blank before one can select another language to translate to.
___

#### Expected behavior

In Wave **v0.16.0**, tables *were compatible* with the Google Translate browser extension. The page is successfully translated and actions can be performed normally, as shown below:

![wave_16](https://user-images.githubusercontent.com/29825155/206808261-eae88e35-e377-4e49-8051-c41e45b812bf.gif)

#### Steps To Reproduce

1. Run `table_groupby_selection.py` below with `Faker==13.3.4` installed. 
(`wave run table_groupby_selection.py`)
   1. Run with `h2o-wave==0.16.0`
   2. Run with `h2o-wave==0.22.0` and/or `h2o-wave==0.24.1`

`table_groupby_selection.py`:
```python
# Table / Group by
# Allow grouping a table by column values.
# #table
# ---
import random
from faker import Faker
from h2o_wave import main, app, Q, ui

fake = Faker()

_id = 0


class Issue:
    def __init__(self, text: str, status: str, progress: float, icon: str, notifications: str):
        global _id
        _id += 1
        self.id = f'I{_id}'
        self.text = text
        self.status = status
        self.views = 0
        self.progress = progress
        self.icon = icon
        self.notifications = notifications


# Create some issues
issues = [
    Issue(
        text=fake.sentence(),
        status=('Closed' if i % 2 == 0 else 'Open'),
        progress=random.random(),
        icon=('BoxCheckmarkSolid' if random.random() > 0.5 else 'BoxMultiplySolid'),
        notifications=('Off' if random.random() > 0.5 else 'On')) for i in range(100)
]

# Create columns for our issue table.
columns = [
    ui.table_column(name='text', label='Issue'),
    ui.table_column(name='status', label='Status'),
    ui.table_column(name='notifications', label='Notifications'),
    ui.table_column(name='done', label='Done', cell_type=ui.icon_table_cell_type()),
    ui.table_column(name='views', label='Views'),
    ui.table_column(name='progress', label='Progress', cell_type=ui.progress_table_cell_type()),
]


@app('/demo')
async def serve(q: Q):
    if q.args.show_inputs:
        q.page['form'].items = [
            ui.text(f'selected={q.args.issues}'),
            ui.button(name='show_form', label='Back', primary=True),
        ]
    else:
        q.page['form'] = ui.form_card(box='1 1 -1 11', items=[
            ui.table(
                name='issues',
                columns=columns,
                rows=[ui.table_row(
                    name=issue.id,
                    cells=[issue.text, issue.status, issue.notifications, issue.icon, str(issue.views),
                        str(issue.progress)]) for issue in issues],
                groupable=True,
                values=['']
            ),
            ui.button(name='show_inputs', label='Submit', primary=True)
        ])
    await q.page.save()
```

`requirements.txt`:
```
Faker==13.3.4
```

Would it be possible to please get compatibility with the Google Translate browser extension back for tables? 

This would allow us to upgrade to the latest version of Wave and easily keep working with different languages in tables.

Thank you! Really appreciate the support on these table issues.

**Comment by mturoci** [comment]
Hi @chelseanbr. The google translate worked by accident TBH since we didn't take it into mind while developing at all. Will classify this as a feature then. Thanks for bringing this up!

**Comment by mturoci** [comment]
@chelseanbr can you please try against the latest Wave? Seems to be working fine for me.

https://user-images.githubusercontent.com/64769322/208679851-12a6b31d-6353-4d42-a1e6-ef8a11a20631.mov

Also what's your Google Translate extension version?

**Comment by mturoci** [comment]
Closing for lack of info. Feel free to reopen.

**Comment by chelseanbr** [comment]
Hi @mturoci, sorry for the late reply! I can confirm the issue was indeed fixed with the latest Wave v0.24.2. Really appreciate your help!
